packages/utils/src/formatters/formatDataForMutation.js

```diff
@@ -0,0 +1,136 @@
+//@flow
+import * as R from 'ramda';
+
+import { MUTATION_TYPE } from '../constants';
+import * as selectors from '../selectors';
+import * as verifiers from '../verifiers';
+import type { MutationType, FieldSchema, TableSchema, Schema } from '../types';
+
+const formatFieldDataForCreateMutation = (tableSchema: TableSchema, schema: Schema) => (data: Object, fieldName: string) => {
+  const fieldSchema: ?FieldSchema = selectors.getFieldSchemaByName(fieldName, tableSchema);
+
+  let formatedFieldData = data;
+
+  if (fieldSchema) {
```

Comment:
It's very complicated sequence of `if-else`.
What do you think about rewrite this code with `R.cond`? Reduce number of padded blocks.

```
formatedFieldData = R.cond([
  [R.both(isRelation, isListField), ...],
  [R.both(isRelation, R.not(isListField), ...],
  ...
])(fieldSchema);
```